packages/vuetify/src/components/VTextField/VTextField.tsx

```diff
@@ -151,6 +151,12 @@ export const VTextField = genericComponent<Omit<VInputSlots & VFieldSlots, 'defa
         })
       }
     }
+    function onKeydown (e: KeyboardEvent) {
+      if (props.type.toLowerCase() === 'number' &&
```

Comment:
Probably all input keyDown events should stop bubbling up. 

`e.stopPropagation()` should be enough